opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -2331,10 +2510,10 @@ namespace Opm {
                     const std::string msg = "Compute initial well solution for " + well->name() + " initially failed. Continue with the previous rates";
                     deferred_logger.warning("WELL_INITIAL_SOLVE_FAILED", msg);
                 }
+                // If we're using local well solves that include control switches, they also update
+                // operability, so reset before main iterations begin
+                well->resetWellOperability();
```

Comment:
Is moving this to be inside the ` if (param_.solve_welleq_initially_ && well->isOperableAndSolvable()) {` intentional or it is something `git` misunderstood?

Reply:
This part of the code I intend not to change at all. I assume it is me to blame rather than git :) . It is fixed now.